CONTRIBUTING.md

```diff
@@ -169,36 +171,36 @@ Follow these steps to start contributing ([supported Python versions](https://gi
    $ pip install -e .
    ```
 
-   If you have already cloned that repo, you might need to `git pull` to get the most recent changes in the `datasets`
+   If you have already cloned `datasets`, you might need to `git pull` to get the most recent changes in the `datasets`
    library.
    
-   Depending on your OS, you might need to install some external libraries, as well, if the `pip` installation fails.
+   Depending on your OS, you may need to install some external libraries as well if the `pip` installation fails.
    
-   For macOS, you will likely need [MeCab](https://taku910.github.io/mecab/), which can be installed from Homebrew:
+   For macOS, you will likely need [MeCab](https://taku910.github.io/mecab/) which can be installed from Homebrew:
    
    ```bash
    brew install mecab
    ```
 
 5. Develop the features on your branch.
 
-   As you work on the features, you should make sure that the test suite
-   passes. You should run the tests impacted by your changes like this:
+   As you work on your code, you should make sure the test suite
+   passes. Run the tests impacted by your changes like this:
 
    ```bash
    $ pytest tests/<TEST_TO_RUN>.py
    ```
 
    You can also run the full suite with the following command, but it takes
-   a beefy machine to produce a result in a decent amount of time now that
-   Transformers has grown a lot. Here is the command for it:
+   a powerful machine to generate the results in a tolerable amount of time now that
+   🤗 Transformers has grown a lot. Here is the command for it:
```

Comment:
At this stage even a powerful machine would take too long. I would remove this entirely.